### PR TITLE
Added more user roles to fakedata

### PR DESCRIPTION
**Pull request by zaher-aa**

### CODE REVIEW

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by RandSohail — APPROVED** [review]

**Review by AymanSami98 — APPROVED** [review]

File: src/database/fakeData/index.ts [hunk 1]
```diff
@@ -64,11 +69,25 @@ const users = [
...
   },
+  {
+    name: 'Master Admin',
+    email: 'masteradmin@gmail.com',
+    password: '$2b$10$O2cKyz8IAgx0jI2OO89j0uZWX9avosPQQ3xWZzcQn7fP/Y3OeFfXG',
+    user_role_id: 2,
+    user_status_id: 2,
+    acc_paid_revenue: 0,
+    free_to_be_paid_revenue: 0,
+    image: 'data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==',
```
Comment (Jood80) [inline src/database/fakeData/index.ts:80]: Nope. this image value will be the uploaded string coming from `s3` service. Remember, the `updateUser` DTO will have the image in `base46` but we won't store it in the database like that

File: src/database/fakeData/index.ts [hunk 2]
```diff
@@ -15,13 +15,18 @@ const userRoles = [
     updated_at: new Date(),
     created_at: new Date(),
   },
+  {
+    name: 'master_admin',
```
Comment (Jood80) [inline src/database/fakeData/index.ts:19]: Is the `master_admin` the same as `system_admin` or `admin`?

**Review by Jood80 — CHANGES_REQUESTED** [review]

File: src/database/fakeData/index.ts (hunk 1 above)
Comment (Jood80) [reply src/database/fakeData/index.ts:80]: Add this picture for instance 😁 
https://nextup-comedy-bucket-gfs.s3.eu-west-2.amazonaws.com/2/1660034083405.jpeg

Comment (zaher-aa) [reply src/database/fakeData/index.ts:80]: Ok, its a nice picture 😇

File: src/database/fakeData/index.ts (hunk 2 above)
Comment (zaher-aa) [reply src/database/fakeData/index.ts:19]: Yes its the same, the founders named it `master_admin`

**Review by Jood80 — APPROVED** [review]

**Review by Jood80 — APPROVED** [review]

**Pull request merged**